### ISSUE TITLE
Add Entra Admins to PostgreSQL Flexible Server?

### ISSUE THREAD

**Issue by nraooptum** [Azure/azure-service-operator#4256]
We use Workload Identities (managed identity, essentially) to connect our apps to PostgreSQL Flexible Server. Is there a way to add Entra admins to the server? Essentially, the equivalent of terraform's `azurerm_postgresql_flexible_server_active_directory_administrator`. I saw [User](https://azure.github.io/azure-service-operator/reference/dbforpostgresql/v1/#dbforpostgresql.azure.com/v1.User) but that seems to be for SQL auth users, not Entra identities.

**Comment by matthchr** [comment]
I thought that there was but now that I'm looking I don't see it. We'll take a look at adding this

**Comment by nraooptum** [comment]
Awesome, thanks! Looking forward to it. We've been doing some prototyping with this to see if we can switch parts of our enterprise to a new deployment model (Azure Service Operator v2 + CDK8s) and bundling our infrastructure artifacts as an OCI artifact using ORAS (stored in ACR, deployed along with our microservices via ArgoCD), and so far it's all looking very promising.

**Comment by matthchr** [comment]
Moving this to 2.11 as I don't have time to finish it in 2.10.

**Comment by nraooptum** [comment]
Thank you for the update!

**Comment by Paul-Brauckmann** [comment]
Any updates on this?

**Comment by ubikusss** [comment]
Any update on this feature?

**Comment by maggix** [comment]
👋 Just checking in—is this feature implemented or it still on the roadmap? Very interested here! Thanks!

**Comment by Xangliev** [comment]
It's quite interesting that you can enable [threat protection](https://azure.github.io/azure-service-operator/reference/dbforpostgresql/v1api20240801/#FlexibleServersAdvancedThreatProtectionSettings) on servers, which will in turn advice users to disable password authentication in favor of MS Entra authentication. But ASO doesn't support the API to manage entra administrators

**Comment by matthchr** [comment]
Yes this is still on the roadmap. We had gotten somewhat sidetracked from finishing this but I see that there is quite a bit of interest. I will try to pick this up again for the next release.
